google-cloud-bigtable/Rakefile

```diff
@@ -174,6 +174,34 @@ task :doctest do
   sh "bundle exec yard config load_plugins true && bundle exec yard doctest"
 end
 
+desc "Run rubocop on yard examples."
+task :rubocop_yard_examples do
+  YARD.parse "lib/**/*.rb"
+  registry = YARD::Registry.load_all
+  examples = registry.all.map { |object| object.tags(:example) }.flatten
+
+  dir_name = "rubocop_yard_examples"
+  rm_rf dir_name if File.exist? dir_name
```

Comment:
nit: `if File.exist?` is unnecessary since `rm_rf` will just ignore nonexistent paths.

Reply:
Fixed, thanks!